### PR TITLE
Updating Readme

### PR DESCRIPTION
**Pull request by bitcoinshirt**
I wanted to add a bit more information in the readme. 
Tried to provide as much information about the software concisely.
Added "Features" section for a quick overview for people to know what to expect.
Feel free to make any edits, this is my pull request, and not even sure I'm doing it right. Or just disregard it, no harsh feelings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,21 +8,38 @@
 
 ## Introduction 
 
-BTCPay Server is an Open Source payment processor, written in C#, that conforms to the invoice API of [Bitpay](https://bitpay.com/).
-This allows easy migration of your code base to your own, self-hosted payment processor.
+BTCPay Server is a free and open-source cryptocurrency payment processor which allows you to receive payments in Bitcoin and altcoins directly, with no fees, transaction cost or a middleman.
 
-This solution is for you if:
+BTCPay is a non-custodial invoicing system which eliminates the involvement of a third-party. Payments with BTCPay go directly to your wallet, which increases the privacy and security. Your private keys are never uploaded to the server. There is no address re-use, since each invoice generates a new address diriving from your xpub key.
```
Comment (ch4ot1c) [inline README.md]: `s/diriving/deriving/`, `xpubkey` one word

File: README.md [hunk 2]
```diff
@@ -8,21 +8,38 @@
...
 
-* You are currently using Bitpay as a payment processor but are worried about their commitment to Bitcoin in the future
-* You want to be in control of your own funds
-* Bitpay compliance team decided to reject your application
-* You want lower fees (we support Segwit)
-* You want to become a payment processor yourself and offer a BTCPay hosted solution to merchants
-* You want a way to support currencies other than those offered by Bitpay
+The software is built in C# language and conforms to the invoice API of BitPay. It allows easy migration of your code base to your own, self-hosted payment processor.
```
Comment (ch4ot1c) [inline README.md]: Just `in C#`; link `the invoice API of Bitpay` to https://bitpay.com/api; final sentence could be 'It allows for your website to be easily configured as a self-hosted crypto payment processor'.

File: README.md [hunk 3]
```diff
@@ -8,21 +8,38 @@
...
+* Lightning Network support (LND and c-lightning)
+* Altcoin support
+* Complete control over private keys
+* Fully compatability with BitPay API (easy migration)
+* Enchanced Privacy
+* SegWit support
+* Process payments for others
+* Payment Buttons
+* Point of Sale
+* No transaction fees
```
Comment (ch4ot1c) [inline README.md]: (other than those for the crypto networks)

File: README.md [hunk 4]
```diff
@@ -8,21 +8,38 @@
...
+
+* Direct, P2P Bitcoin payments
+* Lightning Network support (LND and c-lightning)
+* Altcoin support
+* Complete control over private keys
+* Fully compatability with BitPay API (easy migration)
+* Enchanced Privacy
+* SegWit support
+* Process payments for others
+* Payment Buttons
```
Comment (ch4ot1c) [inline README.md]: lowercase `b`

File: README.md [hunk 5]
```diff
@@ -8,21 +8,38 @@
...
+Thanks to the apps built on top of it, you can use BTCPay to receive donations or have an in-store POS system.
+
+## Features
+
+* Direct, P2P Bitcoin payments
+* Lightning Network support (LND and c-lightning)
+* Altcoin support
+* Complete control over private keys
+* Fully compatability with BitPay API (easy migration)
+* Enchanced Privacy
```
Comment (ch4ot1c) [inline README.md]: lowercase `p`

File: README.md [hunk 6]
```diff
@@ -8,21 +8,38 @@
...
-* You are currently using Bitpay as a payment processor but are worried about their commitment to Bitcoin in the future
-* You want to be in control of your own funds
-* Bitpay compliance team decided to reject your application
-* You want lower fees (we support Segwit)
-* You want to become a payment processor yourself and offer a BTCPay hosted solution to merchants
-* You want a way to support currencies other than those offered by Bitpay
+The software is built in C# language and conforms to the invoice API of BitPay. It allows easy migration of your code base to your own, self-hosted payment processor.
 
-## We support altcoins!
+You can run BTCPay as a self-hosted solution on your own server, or use a third-party host.
```
Comment (ch4ot1c) [inline README.md]: Link to third-party hosts

**Comment by NicolasDorier** [comment]
awesome, let me know when you fixed the nits of @ch4ot1c , will merge.

**Comment by bitcoinshirt** [comment]
@ch4ot1c Thanks for suggesting the changes. I've updated the document now.
@NicolasDorier Great, all done.

If you have further suggestions, let me know.

**Pull request merged**